index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="fill-100">
+    <nav class="navigation">
+      <span>Logo</span>
```

Comment:
- [ ] Please take note that the logo suppose to have a link (a tag). This is for three reasons: convention, semantics, and user experience(UX). 👍